Assets/HoloToolkit/Utilities/Editor/SceneLauncherEditor.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.SceneManagement;
+
+namespace HoloToolkit.Unity
+{
+    [CustomEditor(typeof(SceneLauncher))]
+    public class SceneLauncherEditor : Editor
```

Comment:
What is the purpose for having a special UI and list of scenes in the SceneLauncher? Previously the list of scenes was automatically generated from the list of scenes the user specified via the built-in Unity scene list in Build Settings. With this change there will be two mostly-equivalent lists and UIs.